import-service/app-engine.tf

```diff
@@ -0,0 +1,26 @@
+
+resource "google_app_engine_application" "gae_import_service" {
+  project     = module.import-service-project.project_name
+  location_id = "us-central"
+}
+
+# firewall rules that allow broad ips to access GAE
+# This doesn't work, see https://github.com/terraform-providers/terraform-provider-google/issues/5681
```

Comment:
This is going to be a problem. I can write a script to create the firewall rules, I guess.

Reply:
update: i did